server/repositories/cmsApi.js

```diff
@@ -1,37 +1,70 @@
-const { getEstablishmentUiId } = require('../utils');
+/* eslint-disable class-methods-use-this */
+const { DrupalJsonApiParams: Query } = require('drupal-jsonapi-params');
+
+const topicsQuery = new Query()
+  .addFields('taxonomy_term--tags', [
+    'drupal_internal__tid',
+    'name',
+    'description',
+  ])
+  .addFields('taxonomy_term--moj_categories', [
+    'name',
+    'description',
+    'field_legacy_landing_page',
+  ])
+  .addFilter(
+    'vid.meta.drupal_internal__target_id',
+    ['moj_categories', 'tags'],
+    'IN',
+  )
+  .addPageLimit(100)
+  .getQueryString();
 
 class CmsApi {
```

Comment:
This is going to grow but don't really want to presuppose a tidier structure until we have some additional queries.

Can envision potentially extracting out some query object which encapsulates the query and deserializing but probably makes sense to do this when we have more examples